simulation/container_interface.py

```diff
@@ -49,19 +48,17 @@ def is_alive(self) -> bool:
         else:
             return False
 
-    def set_population_groups(self, infection_prob: float, infection_range: float,
-                              recover_prob: float, dead_prob: float):
-        self.smember = groups.SGroup(x=0, y=0, infection_prob=infection_prob)
-        self.imember = groups.IGroup(x=0, y=0, infection_range=infection_range,
-                                     recover_prob=recover_prob, death_prob=dead_prob)
-        self.rmember = groups.RGroup(x=0, y=0)
-        self.dmember = groups.DGroup(x=0, y=0)
+    def add_instances(self, instance_amount: int, condition_to_set: str,
+                      infection_probability: float, recover_probability: float,
+                      dead_probability: float, infection_range: float) -> None:
+        for i in range(instance_amount):
+            x = random.uniform(0, self.width - 1)
+            y = random.uniform(0, self.height - 1)
+            self.person = groups.Person(x=0, y=0, infection_probability=infection_probability,
+                                        recover_probability=recover_probability,
+                                        dead_probability=dead_probability, infection_range=infection_range)
+            self.person.x = x
+            self.person.y = y
```

Comment:
Why `x` and `y` are not set in `Person` constructor to these values? Is there some reason?